src/RegistryCoordinator.sol

```diff
@@ -446,6 +446,16 @@ contract RegistryCoordinator is
         ejectionCooldown = _ejectionCooldown;
     }
 
+    /**
+     * @notice Sets the deregistration cooldown, which is the time an operator must wait in 
+     * seconds after fully deregistering before registering for any quorum
+     * @param _deregistrationCooldown the new deregistration cooldown in seconds
+     * @dev only callable by the owner
+     */
+    function setDeregistrationCooldown(uint256 _deregistrationCooldown) external onlyOwner {
+        deregistrationCooldown = _deregistrationCooldown;
+    }
```

Comment:
needs an event.
might want this added to the initializer as well.

I'm considering if any input sanitization makes sense here too -- should this have a min or max? is 0 an OK input? I feel like it's probably OK to not have this, given the owner-only nature of the function... I don't see any super "unsafe" values

Reply:
Didn't add to init because of bytecode size but I will see if it can be squeezed in. I could see really long values being not great for the delay but will probably not add sanitization for the same bytecode size issue

Reply:
discussed in-person, no sanitization seems OK, not in initializer is also acceptable to me (although not my ideal), no event is something I can live with but probably the most desirable of these